snaps/reference/permissions.md

```diff
@@ -216,6 +216,8 @@ Specify this permission in the manifest file as follows:
 }
 ```
 
+#### Allowed origins
+
 Alternatively, you can specify the caveat `allowedOrigins` to restrict requests to specific domains or Snap IDs. 
```

Comment:
Maybe this should say "to restrict **all** requests" to differentiate from the guide we are adding